test/common/services/es_archiver.js

```diff
@@ -30,9 +46,17 @@ export async function EsArchiverProvider({ getService }) {
 
   const dataDir = config.get('esArchiver.directory');
 
-  return new EsArchiver({
+  const esArchiver = new EsArchiver({
     client,
     dataDir,
     log,
   });
+
+  if (config.get('esArchiver.automaticallyDisableAnimations')) {
```

Comment:
Why do we need to do this? Is it not enough to set it in `ui_settings.js`?

Reply:
Well, the esArchiver wipes out the .kibana index on a regular basis, which is why I added the ability for test configs that use the esArchiver to automatically update the uiSettings without needing to call `uiSettings.update()` after each `esArchiver.load()`. I'm going to fix the config name though to not explicitly mention disabling animations, as what it's really doing is enforcing the overrides in the `kibanaServer.uiSettings` service.